src/wallet/rpcwallet.cpp

```diff
@@ -2499,21 +2499,22 @@ static UniValue loadwallet(const JSONRPCRequest& request)
             }.Check(request);
 
     WalletContext& context = EnsureWalletContext(request.context);
-    WalletLocation location(request.params[0].get_str());
+    std::string name(request.params[0].get_str());
```

Comment:
aac063f28e4ed48180c2ba7a51706af969ca1e55, nit
```suggestion
    const std::string name(request.params[0].get_str());
```